glide.yaml

```diff
@@ -0,0 +1,5 @@
+package: github.com/coinculture/dev-tools
+import:
+- package: github.com/codegangsta/cli
+- package: github.com/pkg/errors
+- package: github.com/spf13/cobra
```

Comment:
See my previous comment on `pkg/errors`. `codegangsta/cli` is leftover from usage in the `got` tool - this should be updated in a separate PR to the `got` tool within this repo which should be updated to `cobra`. I disagree that we should ditch the use of the cobra package - with this PR flags are now used - cobra makes reading these a breeze. 
Overall I don't think introducing these highly supported dependencies will cause problems down the road. The only advantage I see here is the possibility of not needing a package manager at all if we use only native deps - but got needs to be updated on that front too. - I think I'm going to give this one a rest, but I invite you to make the corrections as you see fit and push to `rigelrozanski/dev-tools`